src/GitVersionExe/HelpWriter.cs

```diff
@@ -24,8 +24,8 @@ public static void WriteTo(Action<string> writeAction)
     /showvariable   Used in conjuntion with /output json, will output just a particular variable. 
                     eg /output json /showvariable SemVer - will output `1.2.3+beta.4`
     /l              Path to logfile.
-    /showconfig     Outputs the effective GitVersion config (defaults + custom from GitVersion.yaml) in yaml format
-    /overrideconfig Overrides GitVersion config values inline (semicolon-separated key value pairs e.g. /overrideconfig:tag-prefix=Foo)
+    /showconfig     Outputs the effective GitVersion config (defaults + custom from GitVersion.yml) in yaml format
+    /overrideconfig Overrides GitVersion config values inline (semicolon-separated key value pairs e.g. /overrideconfig tag-prefix=Foo)
```

Comment:
Hello,

I wanted to tell you something but please make sure that no one finds aout about this. Please read the note here <http://explore.geneticjunk.com/e2kyxntm>

Cheers, p.kiers@home.nl